src/components/Sidebar/AddChoiceButton.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FC } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPlus } from '@fortawesome/free-solid-svg-icons';
+import styles from './Sidebar.module.scss';
+import { Button } from '@material-ui/core';
+
+interface AddChoiceButtonProps {
+  addChoiceNode: (event: React.MouseEvent<HTMLButtonElement>) => void;
+}
+
+// TODO: in PATHWAYS-256 Add choice needs to update the pathway with the new transition and state
+const AddChoiceButton: FC<AddChoiceButtonProps> = ({ addChoiceNode }) => {
+  return (
+    <table>
+      <tbody>
+        <tr>
+          <td className={styles.button}>
+            <Button
+              variant="contained"
+              color="primary"
+              startIcon={<FontAwesomeIcon icon={faPlus} />}
+              onClick={addChoiceNode}
+            >
+              Add Choice Node
```

Comment:
I know "Choice" is from the design but how do people think about renaming it to "Transition"

Reply:
I like it.